docs/tut-making-jettycat.md

```diff
@@ -529,15 +536,15 @@ Then, open the texture "Button_Play" and make sure that its axis is at the cente
 
 ![The collision shape of a "Play" button](./images/tutJettyCat_28.png)
 
-After that, create a new template with this texture. In its On Step code, put the following:
+After that, create a new template with this texture. In its Frame start code, put the following:
```

Comment:
Can _(and should)_ be replaced with Pointer Click event. These events do not depend on catmods, btw, so they are recommended for general usage when coordinates and other values are not relevant. And if they are, there is always ct.pointer.